src/WebUI/ClientApp/src/app/model/DocumentTemplate.ts

```diff
@@ -0,0 +1,26 @@
+export class DocumentTeplate{
```

Comment:
❓ Why do you have to create these, didn't nswag created it for you ??

Reply:
Yes, it did. This was created before action method was created. Not required anymore. Removed.